src/components/Field.jsx

```diff
@@ -150,6 +150,51 @@ export const DropdownField = translate()((props) => {
   )
 })
 
+class FolderPickerFieldComponent extends Component {
+  constructor (props, context) {
+    super(props)
+    this.store = context.store
```

Comment:
Is linking a component like this one to the store is the best way to achieve data fetching ?
I could be missing a point, but a dropdown looks like a dumb component to me, which should be hydrated with external properties. But I also assume that you tried to externalize dependances to the store and it did not work because of the `StatefulForm`. Am I right ?